v2/src/js/p2p/index.js

```diff
@@ -0,0 +1,110 @@
+const WS = require('libp2p-websockets')
+const Multiplex = require('pull-mplex')
+require('libp2p-crypto-secp256k1') // no need to do anything with this, just require it.
+const SECIO = require('libp2p-secio')
+const Bootstrap = require('libp2p-bootstrap')
+const KadDHT = require('libp2p-kad-dht')
+const libp2p = require('libp2p')
+const mergeOptions = require('merge-options')
+const PeerInfo = require('peer-info')
+const crypto = require('libp2p-crypto').keys
+const PeerId = require('peer-id')
+const TCP = require('libp2p-tcp')
+const util = require('util')
+
+const RoutingDiscovery = require('./discovery')
+
+const bootstrapers = [
+  "/ip4/192.168.2.112/tcp/34001/ipfs/16Uiu2HAm3TGSEKEjagcCojSJeaT5rypaeJMKejijvYSnAjviWwV5",
```

Comment:
this my local run - need to figure out how to set this well, but for now maybe it should be pointing at testnet?